packages/debs/amd64/agent/Dockerfile

```diff
@@ -1,35 +1,52 @@
-FROM debian:7
+FROM debian:10
 
-ENV DEBIAN_FRONTEND noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
 
 # Installing necessary packages
-RUN echo "deb http://archive.debian.org/debian/ wheezy contrib main non-free" > /etc/apt/sources.list && \
-    echo "deb-src http://archive.debian.org/debian/ wheezy contrib main non-free" >> /etc/apt/sources.list && \
-    apt-get update && apt-get install -y --force-yes apt-utils && \
-    apt-get install -y --force-yes \
-    curl gcc make sudo wget expect gnupg perl-base=5.14.2-21+deb7u3 perl \
-    libc-bin=2.13-38+deb7u10 libc6=2.13-38+deb7u10 libc6-dev build-essential \
-    cdbs devscripts equivs automake autoconf libtool libaudit-dev selinux-basics \
-    libdb5.1=5.1.29-5 libdb5.1-dev libssl1.0.0=1.0.1e-2+deb7u20 procps gawk libsigsegv2
-
-RUN apt-get update && apt-get build-dep python3.2 -y --force-yes
-
-RUN curl -OL http://packages.wazuh.com/utils/gcc/gcc-9.4.0.tar.gz && \
-    tar xzf gcc-9.4.0.tar.gz  && cd gcc-9.4.0/ && \
+RUN apt-get update && apt-get install -y build-essential \
+            curl wget git gawk devscripts sudo libssl-dev \
+            unzip zip tar texinfo pkg-config clang-tidy uuid \
+            libmpfr-dev libgmp3-dev libmpc-dev autopoint libtool \
+            zlib1g-dev libgcrypt20-dev libmagic-dev libpopt-dev \
+            libmagic-dev libsqlite3-dev  gettext \
+            ninja-build libsystemd-dev libaudit-dev selinux-basics \
+            libarchive-dev debhelper libssl-dev libbz2-dev \
+            && rm -rf /var/lib/apt/lists/*
+
+RUN echo "deb http://ftp.de.debian.org/debian bullseye main" >>/etc/apt/sources.list && \
+    apt-get update && apt-get install -y g++-10 && rm -rf /var/lib/apt/lists/*
```

Comment:
Just curious, what uses `g++-10`? I wonder because `build-essential` already includes a c++ compiler and then we build `gcc-13` without specifying which compiler if I'm not mistaken.

Reply:
To build `gcc-13`, we had to install `g++-10`. It could be due to some specific dependencies, but this was the quickest solution I found.